tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -401,7 +402,15 @@ bool isFusible(Operation* op) {
 // For some ops (e.g. lmhlo ops), some operands are the output memrefs
 // Thus these operands are supposed to be updated.
 int getNumResultOperands(Operation* op) {
-  if (op->getDialect()->getTypeID() != TypeID::get<lmhlo::LmhloDialect>()) {
+  if (auto customOp = dyn_cast_or_null<lmhlo_disc::CustomCallOp>(op)) {
+    // TODO(disc): add a registration base mechanism to support custom call op
+    // with more than one results.
+    return 1;
+  }
+
+  if (op->getDialect()->getTypeID() != TypeID::get<lmhlo::LmhloDialect>() &&
```

Comment:
why `LmhloDialect` and `LmhloDiscDialect` has not output operands?

Reply:
Note that here is "not equal" ^v^